yaml/roads.yaml

```diff
@@ -94,6 +91,12 @@ global:
       incline: {col: tags->incline}
       trail_visibility: {col: tags->trail_visibility}
       sac_scale: {col: tags->sac_scale}
+  - &ne_properties
+      name: {col: name}
+      namealt: {col: namealt}
```

Comment:
Except for 3 records (Sunshine Coast Highway, Seward Hwy., and Portage Glacier Hwy.) these are all additional shield refs and primarily E-roads (about 250 total).